server/repository/src/migrations/v2_05_00/mod.rs

```diff
@@ -0,0 +1,44 @@
+use super::{version::Version, Migration, MigrationFragment};
+
+mod add_feedback_form_table;
+
+use crate::StorageConnection;
+
+pub(crate) struct V2_05_00;
+
+impl Migration for V2_05_00 {
+    fn version(&self) -> Version {
+        Version::from_str("2.5.0")
+    }
+
+    fn migrate(&self, _connection: &StorageConnection) -> anyhow::Result<()> {
+        Ok(())
+    }
+
+    fn migrate_fragments(&self) -> Vec<Box<dyn MigrationFragment>> {
+        vec![Box::new(add_feedback_form_table::Migrate)]
+    }
+}
+
+#[cfg(test)]
+#[actix_rt::test]
+async fn migration_2_05_00() {
+    use v2_04_00::V2_04_00;
+
+    use crate::migrations::*;
+    use crate::test_db::*;
+
+    let previous_version = V2_04_00.version();
```

Comment:
Just need to update this, the previous version is actually `V2_04_01`, so that is what we should test upgrading from 🙏

Reply:
Done!